src/processes/helper/sendTopicAutoMessage.js

```diff
@@ -33,12 +33,13 @@ const followTopicServiceQueue = async (userId, topicId, communityMessageFormatId
   }
 };
 const sendTopicAutoMessage = async (userId, topicIds) => {
+  console.log('::: sendTopicAutoMessage when register :::', userId);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While it's good to have logging for debugging, it's important to ensure that sensitive information such as user IDs are not logged in production environments. This could lead to potential security risks. Consider using a conditional statement to only log this information in development environments.

```diff
- console.log('::: sendTopicAutoMessage when register :::', userId);
+ if (process.env.NODE_ENV === 'development') {
+   console.log('::: sendTopicAutoMessage when register :::', userId);
+ }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->